### DISCUSSION TITLE
Where Do I Get My "fnkey.pem"?

### DISCUSSION THREAD

**Discussion by NitroShellMKDS** [RinLovesYou/Flipnote-Signer#11] [Q&A] [answered]
People Have Said It's In The Rom, I Have It On My 3DS And Can't Find "fnkey.pem" In The .nds File...

**Comment by RinLovesYou** [comment, marked as answer]
Because of potential abuse on services like sudomemo i will not get into how to obtain it.

The file name will obviously not be present in the rom like this.
